--- origsrc/libmatemixer-1.10.0/backends/alsa/Makefile.am	2015-02-24 12:47:32.000000000 -0600
+++ src/libmatemixer-1.10.0/backends/alsa/Makefile.am	2015-09-24 10:39:34.335995000 -0500
@@ -37,6 +37,7 @@ libmatemixer_alsa_la_SOURCES =
         alsa-types.h
 
 libmatemixer_alsa_la_LIBADD =                                   \
+	$(top_builddir)/libmatemixer/libmatemixer.la		\
         $(GLIB_LIBS)                                            \
         $(ALSA_LIBS)
 
--- origsrc/libmatemixer-1.10.0/backends/null/Makefile.am	2015-02-24 12:47:32.000000000 -0600
+++ src/libmatemixer-1.10.0/backends/null/Makefile.am	2015-09-24 10:39:38.707245000 -0500
@@ -13,7 +13,9 @@ libmatemixer_null_la_SOURCES =
         null-backend.c                                          \
         null-backend.h
 
-libmatemixer_null_la_LIBADD = $(GLIB_LIBS)
+libmatemixer_null_la_LIBADD = 					\
+	$(top_builddir)/libmatemixer/libmatemixer.la		\
+	$(GLIB_LIBS)
 
 libmatemixer_null_la_LDFLAGS =                                  \
         -avoid-version                                          \
--- origsrc/libmatemixer-1.10.0/backends/oss/Makefile.am	2015-02-24 12:47:32.000000000 -0600
+++ src/libmatemixer-1.10.0/backends/oss/Makefile.am	2015-09-24 10:39:43.030492300 -0500
@@ -28,6 +28,7 @@ libmatemixer_oss_la_SOURCES =
         oss-types.h
 
 libmatemixer_oss_la_LIBADD =                                    \
+	$(top_builddir)/libmatemixer/libmatemixer.la		\
         $(GLIB_LIBS)                                            \
         $(OSS_LIBS)
 
--- origsrc/libmatemixer-1.10.0/backends/pulse/Makefile.am	2015-02-24 12:47:32.000000000 -0600
+++ src/libmatemixer-1.10.0/backends/pulse/Makefile.am	2015-09-24 10:39:46.613697200 -0500
@@ -58,6 +58,7 @@ libmatemixer_pulse_la_SOURCES =
         pulse-types.h
 
 libmatemixer_pulse_la_LIBADD =                                  \
+	$(top_builddir)/libmatemixer/libmatemixer.la		\
         $(GLIB_LIBS)                                            \
         $(PULSEAUDIO_LIBS)
 
